docs/core-concepts/core-sdk/cli/dotnet.md

```diff
@@ -0,0 +1,79 @@
+dotnet command
+==============
+
+## NAME
+
+dotnet -- general driver for running the command-line commands
+
+## SYNOPSIS
+
+dotnet [--version] [--help] [--verbose] [--info] < command > [< args >]
+
+## DESCRIPTION
+dotnet is a generic driver for the CLI toolchain. Invoked on its own, it will give out brief usage instructions. 
+
+Each specific feature is implemented as a command. In order to use the feature, it is specified after dotnet, i.e. `dotnet compile`. All of the arguments following the command are command's own arguments.  
+
+
+## OPTIONS
+`-v, --verbose`
+
+Enable verbose output.
+
+`--version`
+
+Print out the version of the CLI tooling
+
+`--info`
+
+Print out more detailed information about the CLI tooling, such as the current operating system, commit SHA for the 
```

Comment:
Suggestion: such as the current operating system or the commit SHA (Secure Hash Algorithm) for the version.